src/index.js.flow

```diff
@@ -419,6 +419,11 @@ export type DAOConfig<TEntity, TEntityMutator> = {
 };
 
 declare export class DAOResult<TEntity> {
+  constructor(
```

Comment:
yeah, parser expects: `methodName(args: Args): ReturnType;` otherwise it throws error.